e2e/bitrise.yml

```diff
@@ -94,6 +94,73 @@ workflows:
               fi
             done
 
+
+  test_export_output_variables_based_on_patterns:
+    before_run:
+    - _setup
+    steps:
+    - path::./:
+        title: Execute step
+        inputs:
+        - verbose: true
+        - artifact_sources: "*"
+        - export_map: |-
+              *.txt - TXT_FILES
```

Comment:
Wouldn't this be easier to use as a simple map?

```
- export_map:
        TXT_FILES: "*.txt"
        APK_FILES: "*.apk"
```

Reply:
I was thinking about that in my proposed solution the env var and pattern pairs are adhering to the yaml standard as this way the export map will be a list element which is an object with key-value pairs.
Your solution uses a multi-line raw text notation for yaml which is not checked by the yaml linter and may be harder to parse with our user's eyes.
I only reversed the order of pattern-env pairs, because the env names can be yaml keys while the patterns can contain special characters which wouldn't fit in as yaml keys.

What do you think?